### PR TITLE
Update dialog and button

### PR DESCRIPTION
**Pull request by kirill-kruchkov**
## Purpose

* Add IconButton (moved from `app-service`)
* Update Button (rounded corners)
* Update Dialog (rounded corners, close button option)
* Update Dialog docs (add body-only variant with close button)
* Add some icons
* Use `500` instead of `bold` for bold type

### Visualization

image

image

image

### Checklist
- [x] Update changelog

### CODE REVIEW

File: src/components/Dialog/Dialog.md [hunk 1]
```diff
@@ -63,37 +63,16 @@ Goes with normal paddings and auto width on desktop.
...
-      Ik snap het!
-    </Button>
-    <Button onClick={() => {}}>Nee!</Button>
-  </Dialog.Footer>
-</Dialog>
-```
+Body-only dialogs can be used for modals with complex structure (e.g. to display product details).
 
-### Custom body
+Also you can use `shorCloseButton` property to display a cross button in the top right corner.
```
Comment (dinkra) [inline src/components/Dialog/Dialog.md]: Misspelling in `shorCloseButton`.

Comment (kirill-kruchkov) [reply src/components/Dialog/Dialog.md]: Oww thanks!

File: src/animations/slideIn.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import { keyframes } from 'styled-components';
 
-const slideIn = distance => keyframes`
-  0% { transform: translateX(${distance}); opacity: 0; }
-  100% { transform: translateX(0); opacity: 1;}
+const slideIn = (distanceX, distanceY = 0) => keyframes`
```
Comment (frontalicious-martijn) [inline src/animations/slideIn.js]: Could you add JSDoc documentation for the arguments that can be provided? Same for the other animation functions.

File: src/components/Button/__snapshots__/Button_snapshot.spec.js.snap [hunk 3]
```diff
@@ -25,7 +25,7 @@ ShallowWrapper {
     "props": Object {
       "appearance": "primary",
       "children": "Push Me",
-      "className": "Button-hYXUXp eOQLVa",
+      "className": "Button-hYXUXp dxXNXK",
```
Comment (frontalicious-martijn) [inline src/components/Button/__snapshots__/Button_snapshot.spec.js.snap]: If we add `jest-styled-components` to the tests the generated className will stay the same for the most part, maybe worth adding?
https://github.com/styled-components/jest-styled-components#snapshot-testing

File: src/globals/icons/error.svg [hunk 4]
```diff
@@ -0,0 +1,36 @@
+  <svg
```
Comment (frontalicious-martijn) [inline src/globals/icons/error.svg]: You can (in this case) save 46% in size if you put the code through `svgomg` first :-)
https://jakearchibald.github.io/svgomg/

Comment (kirill-kruchkov) [reply src/globals/icons/error.svg]: Oh nice spot! I prefer `svgo` in terminal though :)

File: src/animations/slideIn.js (hunk 2 above)
Comment (kirill-kruchkov) [reply src/animations/slideIn.js]: Yep! I'll create a ticket to do this

File: src/components/Button/__snapshots__/Button_snapshot.spec.js.snap (hunk 3 above)
Comment (kirill-kruchkov) [reply src/components/Button/__snapshots__/Button_snapshot.spec.js.snap]: Yep this makes sense, I'll create a ticket for this as well

File: src/globals/icons/error.svg (hunk 4 above)
Comment (kirill-kruchkov) [reply src/globals/icons/error.svg]: Done

**Pull request merged**